karma.conf.js

```diff
@@ -269,14 +269,6 @@ let customLaunchers = {
     deviceName: 'iPad Pro (9.7 inch) Simulator',
     deviceOrientation: 'portrait',
   },
-  sl_android_44: {
```

Comment:
https://wiki.saucelabs.com/display/DOCS/2019/03/07/Announcing+End+of+Life+for+old+OS+versions+on+Sauce+emulators+and+simulators